qcodes/instrument/parameter.py

```diff
@@ -1686,18 +1682,18 @@ def combine(*parameters, name, label=None, unit=None, units=None,
     Combine parameters into one sweepable parameter
 
     Args:
-        *parameters (qcodes.instrument.parameter.Parameter): the parameters to
-            combine
-        name (str): the name of the paramter
```

Comment:
In the future please only remove types from the docstrings when they have been included in the function signatures